### PR TITLE
Ccg 1575/statedash 2.0 code updates

### PR DESCRIPTION
**Pull request by jbum**
(To Master)
Numerous updates to support new chart on v2-state-dashboard. Updates some of the shared code, styles, and updates the content on v2-state-dashboard.

### CODE REVIEW

**Review by aaronhans — APPROVED** [review]
great work Jim!

File: src/css/_covid.scss [hunk 1]
```diff
@@ -705,7 +705,8 @@ border: none;
 
 .jumbotron + .container {
   padding-top:1.5rem;
-  padding-bottom:2.5rem;
+  /* this bottom padding causes an undesirable white band on v2-state-dashboard at bottom */
```
Comment (aaronhans) [inline src/css/_covid.scss:708]: This change slightly squishes the bottom margin right above the footer on other pages, seems better with this padding

**Comment by aaronhans** [comment]
Charts at bottom are excellent mobile and desktop sizes

File: src/css/_covid.scss (hunk 1 above)
Comment (jbum) [reply src/css/_covid.scss:708]: OK.  Will need your help removing that white band - not sure why it only shows up on v2-state-dashboard.

**Pull request merged**